Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,7 +9,50 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            int[] newArray = new int[array.Length];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                int position = 0;
+                for (int k = 0; k < array.Length; k++)
+                {
+                    if (array[i] > array[k])
+                    {
+                        position++;
+                    }
+                }
+                bool isSameNumberCheck = true;
+
+                while (isSameNumberCheck)
+                {
+                    if (array[i] == 0)
+                    {
+                        isSameNumberCheck = false;
+                    }
+                    else if (array[i] == newArray[position])
+                    {
+                        position++;
+                    }
+                    else
+                    {
+                        isSameNumberCheck = false;
+                    }
+                }
+
+                newArray[position] = array[i];
+
+            }
+            for (int i = 0; i < newArray.Length; i++)
+            {
+                array[i] = newArray[i];
+            }
```

Comment:
I am not quite convinced that this works for every array. Independent wether this works or not you should provide comments to make your code more understandable! Especially when fancy things are happening.

Reply:
added comments. I did more tests, seems to work every time. Don't know where it might not work.